diff --git a/tests/comment.swift b/tests/comment.swift
index 3ec5eea..e01554c 100644
--- a/tests/comment.swift
+++ b/tests/comment.swift
@@ -3,13 +3,13 @@
     vestibulum elementum felis vitae sodales. Integer at varius dui. Maecenas et suscipit felis, sed cursus
     ante. Aliquam erat volutpat. Aliquam vitae pellentesque dolor. Nunc in luctus tortor, non molestie nibh.
 
-    :param: method      Donec malesuada felis ligula, vitae cursus nibh consectetur nec
-    :param: parameters  Curabitur feugiat arcu magna, sit amet imperdiet est varius quis.
-    :param: options     Donec at consequat risus, in lacinia dui. Suspendisse potenti. Maecenas dignissim
-                        turpis in turpis hendrerit
-    :param: error       Fusce vulputate enim neque, vel mollis neque varius eu. Etiam quis quam sollicitudin,
-                        venenatis ex in, pretium enim. In aliquam diam id gravida fringilla.
-    :param: completion  Curabitur ac tempus sapien. Vivamus interdum ligula id nisi volutpat, et convallis
+    - parameter method:     Donec malesuada felis ligula, vitae cursus nibh consectetur nec
+    - parameter parameters: Curabitur feugiat arcu magna, sit amet imperdiet est varius quis.
+    - parameter options:    Donec at consequat risus, in lacinia dui. Suspendisse potenti. Maecenas dignissim
+                            turpis in turpis hendrerit
+    - parameter error:      Fusce vulputate enim neque, vel mollis neque varius eu. Etiam quis quam
+                            sollicitudin, venenatis ex in, pretium enim. In aliquam diam id gravida fringilla.
+    - parameter completion: Curabitur ac tempus sapien. Vivamus interdum ligula id nisi volutpat, et convallis
     */
     func something() -> Int {
         if true {
